src/connection_string.ts

```diff
@@ -856,6 +856,16 @@ export const OPTIONS = {
       return new Logger('MongoClient', { loggerLevel: value as LoggerLevel });
     }
   },
+  maxConnecting: {
+    default: 2,
+    transform({ name, values: [value] }): number {
+      const maxStalenessSeconds = getUint(name, value);
```

Comment:
Could we maybe rename this variable since it's not a time value but a count value.

Reply:
Yes, definitely - I don't even know how that happened 😅